combiners/SSNet_predictions_notebook.py

```diff
@@ -0,0 +1,553 @@
+#!/usr/bin/env python
```

Comment:
Please verify if this file is similar to SSNet_predictions.py. 
If there are some good tweaks or changes, you can use SSNet_predictions.py.

Reply:
The notebook is similar to SSNet_predictions.py and was mostly used for development/testing. There are also some comments added and minor refactoring (in the case of the list of filenames, I'm not sure if it ended up improving the overall code clarity so I might remove that). I will open another PR that includes these in SSNet_predictions.py; there are also some visualizations that I could move to another script or notebook (depending on which is preferred, I saw both used throughout the repo) if you want to include them.